graphql/database/missions/updateMissionNotification.js

```diff
@@ -0,0 +1,23 @@
+import UserMissionModel from './UserMissionModel'
+
+/**
+ * Records a user's closing the "Mission Completed" notification.
+ *
+ * @param {object} userContext - The user authorizer object.
+ * @param {string} userId - The user's ID.
+ * @param {string} missionId - The mission's ID.
+ */
+const updateMissionNotification = async (userContext, userId, missionId) => {
```

Comment:
sorry I was vague in the ticket.  Let's modify this mutation to either update `acknowledgedMissionComplete` or `acknowledgedMissionStarted` fields on the userMission

Reply:
@jedtan ok wow I see my two tickets were kind of open ended, and I see you have a WIP for when a user accepts a mission with the additional logic for moving a user to accepted in the missionDocument.  Up to you if you want to combine the PRs and mutations or keep them seperate.

Reply:
Eh I think separate is okay. They do pretty different things imo